_posts/2020-07-29-parallelized-sampling/parallelized-sampling.Rmd

```diff
@@ -0,0 +1,139 @@
+---
+title: "Parallelized sampling using exponential variates"
+description: > 
+  How can the seemingly iterative process of weighted sampling without replacement be transformed into something highly parallelizable? Turns out a well-known technique based on exponential variates accomplshes exactly that.
+author:
+  - name: Yitao Li
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+bibliography: bibliography.bib
+date: 07-29-2020
+categories:
+  - Probabilistic ML/DL
+  - Concepts
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/dice.jpg
+header-includes:
+   \usepackage{algorithm2e}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+As part of my recent work to support weighted sampling of Spark dataframes in `sparklyr`, I embarked on a journey searching for algorithms that can perform weighted sampling, especially sampling without replacement, in efficient and scalable ways within a distributed cluster-computing framework such as Apache Spark.
```

Comment:
I think you want to use the royal we across the entire blog post, as in:

```
As part of our recent work ..., we embarked.
```

When using the royal we it gives it more credibility, I think, as in, we reviewed your post, etc.

I'll also suggest later on we add a reference to the PR and sparklyr example, such that, for those that who reaelly want to know who to credit this work for, they'll have: a) The citation from distill b) The  author of this blog and c) The actual PR -- Should be quite obvious you are the author whom they should credit even if we use the royal "we".